src/SourceBuild/content/eng/tools/init-build.proj

```diff
@@ -52,6 +52,9 @@
 
     <Copy SourceFiles="@(UnpackedSourceBuildReferencePackages)" DestinationFiles="$(ReferencePackagesDir)%(Filename)%(Extension)" />
 
+    <!-- Delete the original SBRP packages -->
+    <RemoveDir Directories="$(PrebuiltSourceBuiltPackagesPath)SourceBuildReferencePackages" />
```

Comment:
Why do we copy to that location if we then immediately copy to another location and delete this location?

Reply:
Fixed with https://github.com/dotnet/installer/pull/19114/commits/3f02bc95f3815f096425785f6fd3e2a351888a7e